docker-tags/action.yml

```diff
@@ -0,0 +1,49 @@
+name: docker-tags
+
+inputs:
+  version:
+    default: "8.0.2.0"
+    description: "Version string or path to version file to run the check towards"
+  org:
+    default: "jahia"
+    description: "Docker registry (i.e. Docker Hub) organization"
+  repo:
+    default: "jahia-dev-ee"
+    description: "Docker registry (i.e. Docker Hub) repository"
+  username:
```

Comment:
I know it's an improvement and different from the current implementation, but can you replace `username` with `docker_username`, and the same for the password. Incl. in the bash script